docs/commands/securitydata.rst

```diff
@@ -1,3 +1,3 @@
-.. click:: code42cli.cmds.securitydata:security_data
+.. click:: code42cli.cmds.security_data:security_data
```

Comment:
Renamed this because for modules like `departing_employee.py`, we were using a `_` to deliminate where the `-` are in the command name, so just a little consistency improvement...